.travis.yml

```diff
@@ -12,12 +12,10 @@ cache:
 
 php:
   - 5.6
-  - 7.0
-  - 7.1
```

Comment:
irrelevant change?

Reply:
In fact the Travis tests were failing for eZ 1.x, I guessed we could remove that.